### PR TITLE
Fixed build issues

### PR DESCRIPTION
**Pull request by yceruto**

### CODE REVIEW

File: Resources/doc/configuration.rst [hunk 1]
```diff
@@ -56,10 +56,10 @@ There are several configuration options available:
 * ``data_loader`` - name of a custom data loader. Default value: ``filesystem``
   (which means the standard filesystem loader is used).
 * ``controller``
-  * ``filter_action`` - name of the controller action to use in the route loader.
-    Default value: ``liip_imagine.controller:filterAction``
-  * ``filter_runtime_action`` - name of the controller action to use in the route
-    loader for runtimeconfig images. Default value: ``liip_imagine.controller:filterRuntimeAction``
+    * ``filter_action`` - name of the controller action to use in the route loader.
```
Comment (javiereguiluz) [inline Resources/doc/configuration.rst]: This error has also been fixed in #690 but using a different solution.

**Comment by javiereguiluz** [comment]
:+1: nice!

**Comment by makasim** [comment]
@javiereguiluz should I merge this?

**Comment by javiereguiluz** [comment]
I think so. And I'm going to partially revert my previous pull request to avoid conflicts.

**Comment by makasim** [comment]
Thanks @yceruto for the fix and @javiereguiluz  for the review.

**Pull request merged**